### ISSUE TITLE
PGP example not working due to wrong test keys

### ISSUE THREAD

**Issue by karsten-wagner** [mozilla/sops#644]
Following the "[Test with the dev PGP key](https://github.com/mozilla/sops#test-with-the-dev-pgp-key)" example section in the documentation fails, which was very frustrating for me as a first time user.

I found this is due to the repository containing the wrong test keys to decrypt the "example.yaml" file.  
Given key fingerprints in [sops_functional_tests_key.asc](https://github.com/mozilla/sops/blob/master/pgp/sops_functional_tests_key.asc) after importing:

- FBC7B9E2A4F9289AC0C1D4843D16CEE4A27381B4
- D7229043384BCC60326C6FB9D8720D957C3D3074
- B611A2F9F11D0FF82568805119F9B5DAEA91FF86 (public key only)

Used key fingerprints as per example.yaml metadata

- 1022470DE3F0BC54BC6AB62DE05550BC07FB1A0A
- 85D77543B3D624B63CEA9E6DBC17301B491B3F21

Please note that the .sops.yaml is also configured to use a key that has not been provided:

- 1022470DE3F0BC54BC6AB62DE05550BC07FB1A0A

## Steps to reproduce:
```
$ sops --version
sops 3.5.0 (latest)

$ git clone https://github.com/mozilla/sops.git ~/projects/sops
[...] (Current commit at time of writing: https://github.com/mozilla/sops/commit/4bc27f6eb72b1b4090753e9f3dba1d094544e1c3)

$ gpg --import ~/projects/sops/pgp/sops_functional_tests_key.asc
gpg: keybox '/c/API/.gnupg/pubring.kbx' created
gpg: key 3D16CEE4A27381B4: public key "SOPS Functional Tests Key 1 (https://github.com/mozilla/sops/) <secops@mozilla.com>" imported
gpg: key 3D16CEE4A27381B4: secret key imported
gpg: key D8720D957C3D3074: public key "SOPS Functional Tests Key 2 (https://github.com/mozilla/sops/) <secops@mozilla.com>" imported
gpg: key D8720D957C3D3074: secret key imported
gpg: key 3D16CEE4A27381B4: "SOPS Functional Tests Key 1 (https://github.com/mozilla/sops/) <secops@mozilla.com>" not changed
gpg: key D8720D957C3D3074: "SOPS Functional Tests Key 2 (https://github.com/mozilla/sops/) <secops@mozilla.com>" not changed
gpg: key 19F9B5DAEA91FF86: public key "SOPS Functional Tests Key 3 (https://github.com/mozilla/sops/) <secops@mozilla.com>" imported
gpg: Total number processed: 5
gpg:               imported: 3
gpg:              unchanged: 2
gpg:       secret keys read: 2
gpg:  secret keys unchanged: 2
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   1  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 1u

$ sops -d ~/projects/sops/example.yaml
Failed to get the data key required to decrypt the SOPS file.

Group 0: FAILED
  1022470DE3F0BC54BC6AB62DE05550BC07FB1A0A: FAILED
    - | could not decrypt data key with PGP key:
      | golang.org/x/crypto/openpgp error: Could not load secring:
      | open C:\Users\b8zkgf/.gnupg/secring.gpg: The system cannot
      | find the path specified.; GPG binary error: exit status 2

  85D77543B3D624B63CEA9E6DBC17301B491B3F21: FAILED
    - | could not decrypt data key with PGP key:
      | golang.org/x/crypto/openpgp error: Could not load secring:
      | open C:\Users\b8zkgf/.gnupg/secring.gpg: The system cannot
      | find the path specified.; GPG binary error: exit status 2

Recovery failed because no master key was able to decrypt the file. In
order for SOPS to recover the file, at least one key has to be successful,
but none were.
```

**Comment by autrilla** [comment]
Sorry to hear this was a frustrating experience. I think this issue has already been fixed in #591 .

@ajvb or @jvehent, could we cherry-pick that commit into master, so more people aren't confused?

**Comment by ajvb** [comment]
Fixed in master now.
